course3/homework-string/my_string.cpp

```diff
@@ -0,0 +1,209 @@
+#include "my_string.h"
+#include <memory>
+
+        MyString::MyString()
+        {
+            mBuffer=nullptr;
```

Comment:
This doesn't do anything. The smart pointer is initialized to null if default-constructed.